### PR TITLE
minor grammatical and style changes to preface.asc

### PR DESCRIPTION
**Pull request by michaelpeng**
Comma after "It is like any other text editor" to allow the sentence to flow better; same reasoning for comma after "It is much more."  "well-known" instead of "well known". Comma between before the "and" which follows "Coffeescript," in order to break up the sentence. Oxford comma in "...open...[files like computer programs], modify them, and save them again."

Oxford comma in "...customize, extend, and modify."

The "like an elementary student" seems just a bit awkward and perhaps unnecessary, especially since the following "...all the way to..." statement doesn't make another comparison (e.g. "like a pro" or something to that effect). The addition of a comma also makes the sentence easier to read ("...we'll cover everything from the basics..., all the way to...").

### CODE REVIEW

**Comment by 50Wliu** [comment]
Is there any specific reason you changed the apostrophes?  It doesn't seem to make a difference in the final rendered output.

**Comment by michaelpeng** [comment]
Ah that was my mistake, it's definitely supposed to be the original apostrophe. I have no idea how that happened, my bad!

**Comment by michaelpeng** [comment]
Fixed it!

File: book/preface.asc [hunk 1]
```diff
@@ -10,8 +10,8 @@ Pro Git
...
+In the very simplest terms, Atom is a hackable text editor. It is like any other text editor, in that you generally use it to open plain text files like computer programs, modify them, and save them again. It is also much more, in that it’s built on well-known web technologies such as HTML and Coffeescript, and is quite easy to customize, extend, and modify.
 
-In this book, we'll cover everything from the basics that you need to know to edit files like an elementary school student all the way to building brand new Atom-like applications with the shell that underlies Atom. We'll also cover everything in between.
+In this book, we'll cover everything from the basics that you need to know to edit files, all the way to building brand new Atom-like applications with the shell that underlies Atom. We’ll also cover everything in between.
```
Comment (mnquintana) [inline book/preface.asc]: There's another one here

File: book/preface.asc [hunk 2]
```diff
@@ -10,8 +10,8 @@ Pro Git
 
 Welcome to the Flight Manual for Atom, the Hackable Text Editor of the 21st Century.
 
-In the very simplest terms, Atom is a hackable text editor. It is like any other text editor in that you generally use it to open plain text files like computer programs, modify them and save them again. It is also much more in that it's built on well known web technologies such as HTML and Coffeescript and is quite easy to customize, extend and modify.
+In the very simplest terms, Atom is a hackable text editor. It is like any other text editor, in that you generally use it to open plain text files like computer programs, modify them, and save them again. It is also much more, in that it’s built on well-known web technologies such as HTML and Coffeescript, and is quite easy to customize, extend, and modify.
```
Comment (50Wliu) [inline book/preface.asc]: And here :wink: `it's`

**Comment by michaelpeng** [comment]
Oh yeah, I found the third one with find in my text editor as well, haha. That's the weirdest thing, must be some kind of ASCII character quirk?!

**Comment by 50Wliu** [comment]
One final thing: I'm not too sure about the comma addition at `It is like any other text editor, in that...`.  To me, it flows better *without* the comma.

**Comment by michaelpeng** [comment]
Hmm, yeah I see what you mean and I agree. Maybe I should also remove the comma from the second "in that" (immediately following that sentence) just to keep it consistent?

I just pulled from the atom repository so I don't know why it says Pro Git either...oh no. Not being published on atom.io is also kind of a downer haha :fearful: It's just some very, very minor grammatical/style things I noticed so it's not really that big a deal, I suppose T_T

**Comment by michaelpeng** [comment]
Yeah, I just noticed the latest commit: "remove useless preface." It appears that this isn't really a useful document, then...thanks for your help, though!

**Pull request closed without merging**